opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -246,5 +262,88 @@ def end(self):
             self.end_time = util.time_ns()
 
 
+def generate_span_id():
+    """Get a new random span ID.
+
+    Returns:
+        A random 64-bit int for use as a span ID
+    """
+    return random.getrandbits(64)
```

Comment:
We might need to move this into API if we plan to support the "default behavior" in https://github.com/w3c/trace-context/blob/master/spec/20-http_header_format.md#mutating-the-traceparent-field.
We can explore this later, no need to block this PR.

Reply:
Besides what @reyang commented: I remember we kept this private in Java, to not have use this as a defacto standard for creating span ids. Something to consider?

Reply:
I don't have a strong opinion here, and think the bar for making things private is generally a lot lower in java than python. But since we don't have a convention for marking the SDK's "API", hiding everything we don't expect users to call may be a good idea.

That said, if someone is calling into the SDK package instead of just using the API they're already voiding the warranty...